boards/e180-zg120b-tb/doc.txt

```diff
@@ -0,0 +1,199 @@
+/**
+ * @defgroup    boards_e180-zg120b-tb Ebyte E180-ZG120B-TB Test Board
+ * @ingroup     boards
+ * @brief       Support for Ebyte E180-ZG120B-TB Test Board
+
+## Overview
+Ebyte E180-ZG120B Test Board is equipped with the EFM32 microcontroller.
+It is specifically designed for low-power applications, having energy-saving
+peripherals, different energy modes and short wake-up times.
+
+The starter kit is equipped with an Advanced Energy Monitor. This allows you to
+actively measure the power consumption of your hardware and code, in real-time.
+
+## Hardware
+
+### MCU
+| MCU           | EFR32MG1B232F256GM32                                                                    |
+|---------------|-----------------------------------------------------------------------------------------|
+| Family        | ARM Cortex-M4F                                                                          |
+| Vendor        | Ebyte                                                                                   |
+| Vendor Family | EFM32 Mighty Gecko 1B                                                                   |
+| RAM           | 31.0 KiB                                                                                |
```

Comment:
that's an odd number - not 32 k?

Reply:
Same as with the SLTB001a, 1 KiB is reserved for the radio.

Reply:
even if the radio is not used?

Reply:
Then you'll have the 32 KiB available. The Radio blob uses 1 KiB. I'll clarify it.